File: mockmate-backend/src/main/resources/runners/java/runner_int+int.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try {
            List<Integer> list = new ArrayList<>();
            while (sc.hasNext()) {
                String s = sc.next().replace(",", "");
                try {
                    list.add(Integer.parseInt(s));
                } catch(Exception ignored){}
            }
            int a = list.size() > 0 ? list.get(0) : 0;
            int b = list.size() > 1 ? list.get(1) : 0;
            Solution sol = new Solution();
            Object result = sol.{{methodSignature}}(a, b);
            System.out.println(result);
        } catch (Exception e) {
            e.printStackTrace();
            System.exit(1);
        } finally {
            sc.close();
        }
    }
}

{{USER_CODE}}

File: mockmate-backend/src/main/resources/runners/python/runner_int+int.py
import sys
import json

def main():
    input_data = sys.stdin.read().strip()
    if not input_data:
        return

    parts = input_data.replace(',', ' ').split()
    if len(parts) < 2:
        return

    a, b = int(parts[0]), int(parts[1])

    sol = Solution()
    result = sol.{{methodSignature}}(a, b)

    if isinstance(result, bool):
        print(str(result).lower())
    else:
        print(result)

{{USER_CODE}}

if __name__ == '__main__':
    main()
